### PR TITLE
Doc: Make README.md look better :)

### PR DESCRIPTION
**Pull request by yize**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,32 +16,37 @@ You can find the full code for the examples [here](https://github.com/kirjs/reac
...
 ```jsx
-var React = require('react');
-var ReactHighcharts = require('react-highcharts'); // Expects that Highcharts was loaded in the code. 
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+const ReactHighcharts = require('react-highcharts'); // Expects that Highcharts was loaded in the code. 
 
-var config = {
+let config = {
```
Comment (kirjs) [inline README.md]: Why not const?

File: README.md [hunk 2]
```diff
@@ -16,32 +16,37 @@ You can find the full code for the examples [here](https://github.com/kirjs/reac
 To run the demo:
 
  1. run:
+ 
     ```bash
     git clone https://github.com/kirjs/react-highcharts.git
-    npm install
+    npm i
```
Comment (kirjs) [inline README.md]: I would prefer to keep install everywhere to make it more explicit.

**Comment by kirjs** [comment]
Thanks, 2 minor comments, but looks good otherwise

File: README.md (hunk 2 above)
Comment (yize) [inline README.md]: Agree

**Comment by kirjs** [comment]
Thanks, looks better indeed :)

**Pull request merged**